portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase

```diff
@@ -380,6 +380,74 @@ definition {
 			taskCount = "1");
 	}
 
+	@description = "LRQA-53662 Automate LPS-92053 View process item detail page"
+	@priority = "5"
+	test ViewProcessItemDetailPage {
+		property portal.acceptance = "true";
+
```

Comment:
Hi @melodywu 
Would you please add the following line? It is needed to run the tearDown macro. This is a workaround until I organize the scripts better.

property test.name.skip.portal.instance = "WorkflowMetrics#ViewProcessItemDetailPage";

Thank you very much!